iis-common/src/test/java/eu/dnetlib/iis/common/java/jsonworkflownodes/SelectiveTestingConsumer.java

```diff
@@ -41,14 +39,12 @@
  */
 public class SelectiveTestingConsumer implements Process {
 
-    private static final String PROPERTIES_CSV = "expectation_properties_csv";
+    private static final String EXPECTATIONS_CSV = "expectations_csv";
```

Comment:
The thing is `expectations_csv` property value is csv indeed because it supports multiple files when multiple records are verified. `expectation_files_csv` maybe? 

I guess it will be good to use `csv` chunk to indicate mutiple files can be provided and when provided they should be separated with comma.